doc/lua_api.txt

```diff
@@ -3115,7 +3115,15 @@ For the following functions, `v`, `v1`, `v2` are vectors,
 * `vector.cross(v1, v2)`:
     * Returns the cross product of `v1` and `v2`.
 * `vector.offset(v, x, y, z)`:
-    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.
+    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.#
+* `vector.to_string(v)`:
+    * Returns a string `"("..v.x..", "..v.y..", "..v.z..")"`.
+* `vector.from_string(s[, init])`:
```

Comment:
In anticipation of #10324 and #11039, `vector.from_string` should be moved next to `vector.new` since it is a constructor.

Reply:
Ok, done.